src/types.jl

```diff
@@ -9,11 +9,15 @@
 ##  SPDX-License-Identifier: LGPL-3.0-or-later
 ##
 
-const GapObj = @ccall libgap.GAP_DeclareGapObj(:GapObj::Symbol, GAP::Module, Any::Any)::Any
+_declare_gap_obj() = @ccall libgap.GAP_DeclareGapObj(:GapObj::Symbol, GAP::Module, Any::Any)::Any
 
-const SmallBag = @ccall libgap.GAP_DeclareBag(:SmallBag::Symbol, GAP::Module, Any::Any, 0::Cint)::Any
+_declare_bag(sym::Symbol, large::Bool) = @ccall libgap.GAP_DeclareBag(sym::Symbol, GAP::Module, Any::Any, large::Cint)::Any
 
-const LargeBag = @ccall libgap.GAP_DeclareBag(:LargeBag::Symbol, GAP::Module, Any::Any, 1::Cint)::Any
+const GapObj = _declare_gap_obj()::Type
+
+const SmallBag = _declare_bag(:SmallBag::Symbol, false)
+
+const LargeBag = _declare_bag(:LargeBag::Symbol, true)
```

Comment:
```suggestion
const SmallBag = _declare_bag(:SmallBag, false)

const LargeBag = _declare_bag(:LargeBag, true)
```
this is unnecessary and removing it makes it more readable IMO